lib/src/install.rs

```diff
@@ -644,8 +644,16 @@ async fn initialize_ostree_root_from_self(
     }
     f.flush()?;
 
+    let fstab_path = rootfs.join("etc/fstab");
+    state.lsm_label(&fstab_path, "/etc/fstab".into(), false)?;
+
     if let Some(contents) = state.root_ssh_authorized_keys.as_deref() {
-        osconfig::inject_root_ssh_authorized_keys(&root, contents)?;
+        let tmpf_path = osconfig::inject_root_ssh_authorized_keys(&root, contents)?;
+        state.lsm_label(
```

Comment:
I think it'd be better to pass the `state` into this API call and have it perform the labeling, instead of having it return the path.

Reply:
I remember now why I did it the other way, because I was too lazy to update the test /o\ - sorry, I will look at this.